scripts/pull_request_blaster_outer/001_bump_travis_ruby23_to_236.rb

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env ruby
+
+# We're run from the repo's directory
+require 'yaml'
+
+travis = "#{Dir.pwd}/.travis.yml"
+yml = YAML.load_file(travis)
+yml["rvm"].reject! {|r| r.start_with?("2.3")}
+yml["rvm"].unshift("2.3.6")
+
+File.open(travis, 'w') do |file|
+  file.write(YAML.dump(yml).sub("---\n", ""))
```

Comment:
Is there no way to dump yaml without the optional leading `---\n`  ???

Reply:
Good question, but this looks like a reasonable way to do it.